public/SinatraCSS.css

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
One thing I'd suggest is to use all lower-case letters in file names. It doesn't /really/ matter, as most computers these days understand them and can find the files, but it's "the way its done" and it important to not surprise other people (including yourself in 3 months ;) )